test/e2e/fixture/app/context.go

```diff
@@ -17,6 +24,7 @@ type Context struct {
 	destServer             string
 	env                    string
 	parameters             []string
+	jsonnetTlaParameters   []JsonnetParam
```

Comment:
A couple of minor notes:-

- Canonically, these fields are intended to closely reflect the CLI parameters. That's why `parameters` is `[]string` rather than a struct. They can be passed directly to the CLI without modification. 
- Golang is has an unusual standard for abbreviations in names, https://www.google.com/search?q=golang+name+abbreviations&oq=golang+name+abbreviations&aqs=chrome..69i57j69i64.4071j0j7&sourceid=chrome&ie=UTF-8 . I.e. prefer `TLAS` to `Tlas`
- I think you can rename `JsonnetTlaParameter` to just `JsonnetTLA`

Reply:
I think I get it, I will do the changes